.github/workflows/labs-prototypes-ci.yml

```
@@ -11,9 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [20.x]
+        node-version: [21.x]
```

Since 20 is current LTS, I think we should keep testing it. I'd support adding 21 in addition, though. (Probably best for a separate PR, though).

---

Ah I thought it was 21 that introduced the test runner. I shall revert

---

```suggestion
        node-version: [20.x]
```
Ah I thought it was 21 that introduced the test runner. I shall revert